File: labs/step-functions/aws/nodejs/src/format/handler.js
exports.handler = async (event) => {
  const v = event.viacep;
  return {
    status: "SUCCESS",
    cep: event.cep,
    address: {
      street: v.logradouro ?? "",
      complement: v.complemento ?? "",
      neighborhood: v.bairro ?? "",
      city: v.localidade ?? "",
      state: v.uf ?? "",
      ibge: v.ibge ?? "",
    },
    source: "viacep",
    fetchedAt: new Date().toISOString(),
  };
};

File: labs/step-functions/aws/python/src/format/handler.py
from datetime import datetime, timezone


def lambda_handler(event, context):
    v = event["viacep"]
    return {
        "status": "SUCCESS",
        "cep": event["cep"],
        "address": {
            "street": v.get("logradouro", ""),
            "complement": v.get("complemento", ""),
            "neighborhood": v.get("bairro", ""),
            "city": v.get("localidade", ""),
            "state": v.get("uf", ""),
            "ibge": v.get("ibge", ""),
        },
        "source": "viacep",
        "fetchedAt": datetime.now(timezone.utc).isoformat(),
    }
